client/lib/analytics/ad-tracking/ad-track-signup-complete.js

```diff
@@ -159,7 +159,7 @@ export async function adTrackSignupComplete( { isNewUserSite } ) {
 	// Twitter
 
 	if ( mayWeTrackByTracker( 'twitter' ) ) {
-		const params = [ 'track', 'Signup', {} ];
+		const params = [ 'event', 'tw-nvzbs-ode0f' ];
```

Comment:
If we expect this to change, maybe we can move it out to a constant, like we do with Google. If not, this is fine.

Reply:
The docs didn't mention any upcoming changes, so I think this should be stable